frameworks/g-cloud-12/questions/declaration/servicesHaveOrSupportCloudHostingCloudSoftware.yml

```diff
@@ -9,10 +9,9 @@ question: >
 type: radios
 options:
   - label: "Yes"
-  - label: "No"
```

Comment:
Removing the "No" answer makes this an un-failable question. If we're doing that we may as well remove the "No" option from all the questions...

Reply:
We still want the questions to appear in the schema as required fields - it may be that we just remove the `allOf` section for now? However I think it's a useful example of how we'd do this validation in the API eventually.